plugins/org.eclipse.n4js.flowgraphs/src/org/eclipse/n4js/flowgraphs/analysers/ValueNeverUsedAnalyser.java

```diff
@@ -22,10 +22,10 @@
 /**
  * Checks if a local variable is never used.
  */
-public class NeverUsedVisitor extends GraphVisitor {
+public class ValueNeverUsedAnalyser extends GraphVisitor {
```

Comment:
_Value_ is not really a variable right? May be renamed to `VariableNeverUsedAnalyzer`?

Reply:
The comment is wrong. The meaning of this analyser is: It finds e.g. assignments which are obsolete. In other words, the assigned values are never used.
I'll change the comment.